parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestEncryption.java

```diff
@@ -80,11 +80,11 @@ public void test() throws Exception {
     random.nextBytes(columnKey0);
     byte[] columnKey1 = new byte[16];
     random.nextBytes(columnKey1);
-    ColumnEncryptionProperties columnProperties0 = ColumnEncryptionProperties.builder("binary_field", true)
```

Comment:
No changes are needed in this file. The problem is fixed the latest encr branch code.